src/components/CouponDetails/index.jsx

```diff
@@ -969,25 +974,25 @@ class CouponDetails extends React.Component {
                     })}
                     {this.shouldShowSelectAllStatusAlert() && this.renderInfoMessage({
                       message: (
-                        <React.Fragment>
+                        <>
                           {hasAllCodesSelected ? `All ${tableData.count} codes are selected.` : `${selectedCodes.length} codes are selected.`}
                           {!hasAllCodesSelected && (
-                            <Button
-                              variant="link"
-                              className="p-0 pl-1 border-0"
-                              onClick={() => this.setState({
-                                hasAllCodesSelected: true,
-                              })}
-                            >
-                              {`Select all ${tableData.count} codes?`}
-                            </Button>
+                          <Button
+                            variant="link"
+                            className="p-0 pl-1 border-0"
+                            onClick={() => this.setState({
+                              hasAllCodesSelected: true,
+                            })}
+                          >
+                            {`Select all ${tableData.count} codes?`}
+                          </Button>
```

Comment:
nit: Is this indenting correct? It looks like the original indent spacing was previously correct.